docs/units/equivalencies.rst

```diff
@@ -111,8 +111,7 @@ energy can be converted.
 These equivalencies even work with non-base units::
 
   >>> # Inches to calories
-  >>> from astropy.units import imperial
-  >>> imperial.inch.to(imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
+  >>> u.imperial.inch.to(u.imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
```

Comment:
It seems unnecessary to add `from astropy.units import imperial` when it's already available as `u.imperial`.

This:

```
import astropy.units as u
q = 10. * u.imperial.mile
```

is simpler and just as explicit as this

```
import astropy.units as u
from astropy.units import imperial

q = 10. * imperial.mile
```

I don't have a very strong opinion about this, but `u.imperial.mile` also seems more natural to me.